packages/babel-preset-react/src/index.js

```diff
@@ -5,15 +5,28 @@ import transformReactJSXSource from "@babel/plugin-transform-react-jsx-source";
 import transformReactJSXSelf from "@babel/plugin-transform-react-jsx-self";
 
 export default function(context, opts = {}) {
-  const development = opts.development || false;
+  let development = false;
+  let useBuiltIns = false;
+
+  if (opts !== undefined) {
+    if (opts.development !== undefined) development = opts.development;
+    if (opts.useBuiltIns !== undefined) useBuiltIns = opts.useBuiltIns;
+  }
 
   if (typeof development !== "boolean") {
-    throw new Error("Preset react 'development' option must be a boolean.");
+    throw new Error(
+      "@babel/preset-react 'development' option must be a boolean.",
+    );
+  }
+  if (typeof useBuiltIns !== "boolean") {
+    throw new Error(
+      "@babel/preset-react 'useBuiltIns' option must be a boolean.",
+    );
   }
 
   return {
     plugins: [
-      transformReactJSX,
+      [transformReactJSX, { useBuiltIns }],
```

Comment:
`transform-react-jsx` doesn't take `useBuiltIns` at all (what would be its builtin anyway; `Object.assign`? that would only be relevant for the `_jsx` helper at any rate)

It might be good to have a way to pass `pragma`, `pragmaFrag` and `throwIfNamespace` though.

Reply:
https://github.com/babel/babel/blob/9e2828322e8837cbb5b0f1e01a512845f54e56d6/packages/babel-plugin-transform-react-jsx/test/fixtures/useBuiltIns/options.json#L2

it looks like the helper reads the options

https://github.com/babel/babel/blob/9e2828322e8837cbb5b0f1e01a512845f54e56d6/packages/babel-helper-builder-react-jsx/src/index.js#L152-L158